src/main/webapp/scripts/classes/comment-class.js

```diff
@@ -0,0 +1,32 @@
+/*
+ *  Class handles actions related to creating and analyzing new comments
+ *  TODO: add methods to add new comment to youtube video using Youtube API
+ */
+
+class Comment{
+  #MAX_COMMENTS  = 3000;
+  #NO_COMMENT_ERROR = 'No comment inputted';
+  #COMMENT_LENGTH_EXCEEDED_ERROR = 'Comment is too long. We are currently only able to analyze comments of up to approx. 3000 characters.';
+  constructor(text) {
+    this.text = text;
+    this.isValid = !this.getError();
+    this.perspectiveString = this.makePerspectiveRequestString();
+  }
+  getError() {
+    if(this.text == 0) {
```

Comment:
`text` seems like a string type and it is being compared to `0`. Is this intended?

Reply:
I did this to check if the string is empty because non-empty strings are true, but I realized that there are [a lot of ways ](https://stackoverflow.com/questions/154059/how-can-i-check-for-an-empty-undefined-null-string-in-javascript)to do this. I will change it to !this.text because I think that makes more sense to look at than the integer comparison?

Reply:
A common way is to do `!this.text`.

Reply:
I did change it, is the new version not showing up? (I messed up my git history at some point so maybe you'll have to toggle the "Changes from ___ commits" button in the Files changed tab?)